tests/integration/src/test/java/org/mule/test/integration/routing/MessageChunkingTestCase.java

```diff
@@ -87,16 +85,13 @@ public void onNotification(ServerNotification notification) {
     flowExecutionListener.addListener(source -> messagePartsCount.getAndIncrement());
 
     MuleClient client = muleContext.getClient();
-    flowRunner("ObjectReceiver").withPayload(simpleSerializableObject).asynchronously().run();
-    // Wait for the message to be received and tested (in the listener above)
-    assertTrue(chunkingReceiverLatch.await(20L, TimeUnit.SECONDS));
+    flowRunner("ObjectReceiver").withPayload(simpleSerializableObject).run();
```

Comment:
are the changes in this class correct? why remove the latches and meke this synchronous?